features/signal/external/feature.ts

```diff
@@ -0,0 +1,33 @@
+import { Feature } from '@temporalio/harness';
+import * as wf from '@temporalio/workflow';
+import * as assert from 'assert';
+
+const signalData = 'Signaled!';
+const externalSignal = wf.defineSignal<[string]>('externalSignal');
+
+// A workflow that receives a signal and immediately returns with the value of said signal.
+export async function workflow(): Promise<string> {
+  let result = '';
+  wf.setHandler(externalSignal, (str: string) => {
+    result = str;
+  });
+  await wf.condition(() => {
+    return result != '';
+  });
+  return result;
+}
+
+export const feature =
+  !wf.inWorkflowContext() &&
+  new Feature({
+    workflow,
+    execute: async (runner) => {
+      const handle = await runner.executeSingleParameterlessWorkflow();
+      handle.signal(externalSignal, signalData);
```

Comment:
I see I'm late but you should await on this

Reply:
We need a lint rule to catch this in this repo.
I'll add in my feature when I get time to work on it.

Reply:
Good catch, thank you. https://github.com/temporalio/sdk-features/pull/102